### PR TITLE
Task/publish to npm

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -58,8 +58,8 @@ pipeline {
                 container (name: 'kaniko', shell: '/busybox/sh') {
                     sh "#!/busybox/sh\nmkdir -p /root/.docker/"
```
Comment (connoratrug) [inline Jenkinsfile]: maybe add a few line describing why this is happening

**Comment by connoratrug** [comment]
@fdlk 

maybe this is a bit more elegant:
```
rsync -av packages/ docker/dist/ --exclude */node_modules
```

File: packages/scripts/vue.config.js [hunk 2]
```diff
@@ -18,8 +18,6 @@ module.exports = {
     config.externals = {
       'bootstrap': 'bootstrap'
     }
-    config.output.filename = 'js/scripts/[name].js',
```
Comment (connoratrug) [inline packages/scripts/vue.config.js]: lines can be removed , were needed to have /scripts/ subpath

File: packages/scripts/vue.config.js [hunk 3]
```diff
@@ -28,8 +26,8 @@ module.exports = {
   },
   css: {
     extract: {
-      filename: 'css/scripts/[name].css',
-      chunkFilename: 'css/scripts/[name].css'
+      filename: 'css/[name].css',
```
Comment (connoratrug) [inline packages/scripts/vue.config.js]: lines can be removed , sub path no longer needed

File: packages/scripts/vue.config.js (hunk 2 above)
Comment (fdlk) [reply packages/scripts/vue.config.js]: I think otherwise we'll get a chunk hash in the name

**Comment by fdlk** [comment]
> @fdlk
> 
> maybe this is a bit more elegant:
> 
> ```
> rsync -av packages/ docker/dist/ --exclude */node_modules
> ```

I don't think busybox has rsync

File: packages/scripts/vue.config.js (hunk 2 above)
Comment (connoratrug) [reply packages/scripts/vue.config.js]: there is a separate , defidated flag for that

**Pull request merged**